utils/build/docker/ruby/graphql23/app/controllers/system_test_controller.rb

```diff
@@ -8,11 +8,22 @@ def root
   end
 
   def healthcheck
+    version = if defined?(Datadog::VERSION)
+                if Gem.loaded_specs['datadog'].source.is_a?(Bundler::Source::Rubygems)
```

Comment:
I am afraid this will raise a `NoMethodError` if the gem is not loaded:
```ruby
> Gem.loaded_specs['datadog'].source
(irb):1:in `<main>': undefined method `source' for nil (NoMethodError)
```

Reply:
But if Datadog::VERSION is defined, it means that the gem is loaded ?

Reply:
To give more context: We test the library so it should always be loaded. If Datadog::VERSION is not defined, that means that DDTrace::VERSION is defined (and we're on the 1.x branch) If none is defined then the library is not loaded and every tests will fail anyway

Reply:
The logic here is the following:

1. `Datadog` is a namespace which exists in both cases
2. `Datadog::VERSION` exists only when `datadog` gem is loaded and only in new version
3. Otherwise it's `ddtrace` and we know for sure it's also loaded

hence I think we can proceed with those changes.